OsmAnd/res/values/strings.xml

```diff
@@ -289,9 +289,9 @@
     <string name="quick_action_add_destination">Add destination</string>
     <string name="quick_action_replace_destination">Replace destination</string>
     <string name="quick_action_add_first_intermediate">Add first intermediate</string>
-    <string name="quick_action_add_destination_desc">Tap the action button to make your previous destination into an intermediary waypoint to a new destination in the centre of the screen.</string>
-    <string name="quick_action_replace_destination_desc">Center the view away from the destination and onto yourself by tapping the action button.</string>
-    <string name="quick_action_add_first_intermediate_desc">Add an intermediate waypoint before your destination in the middle of the screen by tapping the action button.</string>
+    <string name="quick_action_add_destination_desc">Tap the action button to add a destination at the screen center location. The previously set destination becomes the last intermediate destination.</string>
+    <string name="quick_action_replace_destination_desc">Tap the action button to replace the destination with the screen center location.</string>
+    <string name="quick_action_add_first_intermediate_desc">Tap the action button to add a first intermediate point at the screen center location.</string>
```

Comment:
Very hard to understand this sentence, not sure I even did. These are the questions that need answering, and they are not answered until the end.
1. Why should I tap the action button?
2. Are there any second intermediate points?
3 Intermediate to what?
4. Don't know what "the screen center location" is clear about, but it sounds clear somehow.